services/webdav/pkg/service/v0/service.go

```diff
@@ -52,8 +49,8 @@ var (
 
 // Service defines the extension handlers.
 type Service interface {
-	ServeHTTP(http.ResponseWriter, *http.Request)
-	Thumbnail(http.ResponseWriter, *http.Request)
+	ServeHTTP(w http.ResponseWriter, r *http.Request)
+	Thumbnail(w http.ResponseWriter, r *http.Request)
```

Comment:
Is there a reason for this change?

Not critical - just asking.

Reply:
golinter

![image](https://github.com/owncloud/ocis/assets/1005065/5bee94de-da42-401d-8cd1-af4f69563ceb)